prow/config/config.go

```diff
@@ -1123,7 +1123,8 @@ func validateAgent(v JobBase, podNamespace string) error {
 	k := string(prowapi.KubernetesAgent)
 	b := string(prowapi.KnativeBuildAgent)
 	j := string(prowapi.JenkinsAgent)
-	agents := sets.NewString(k, b, j)
+	p := string(prowapi.TektonAgent)
```

Comment:
Can we validate that we can set Namespace when using pipelines?

Reply:
I fixed the config to allow namespace customisation for `tekton-pipeline` agent, and validated.